From 3852cfd639ebbdd4b340d9bf1ac4b936a4a16dd7 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Sat, 29 Feb 2020 15:13:02 +0000
Subject: [PATCH 40/54] dt-bindings: arm: amlogic: add support for the Tanix
 TX5 Max

The Oranth (Tanix) TX5 Max is based on the Amlogic U200 reference
board with an S905X2 chip.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index e69a8011e605..0216e1d248c4 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -141,6 +141,7 @@ properties:
           - enum:
               - amediatech,x96-max
               - amlogic,u200
+              - oranth,tx5-max
               - seirobotics,sei510
           - const: amlogic,g12a
 
-- 
2.17.1

